ts/sagas/startup/checkProfileEnabledSaga.ts

```diff
@@ -10,17 +10,9 @@ export function* checkProfileEnabledSaga(
   profile: InitializedProfile
 ): IterableIterator<Effect> {
   if (
-    !profile.has_profile ||
-    !profile.email ||
-    !profile.is_inbox_enabled ||
-    !profile.is_webhook_enabled
+    profile.version === 0 &&
```

Comment:
Here we can use `isProfileFirstOnBoarding` selector
Check if we use this raw check somewhere and replace it ;)

Reply:
I noticed that it is a utility check not a real selector, I find out also this other utility check: `hasProfileEmail` I replaced the `!profile.email` with `!hasProfileEmail(profile)` too.

Reply:
the replacements are in 4d8e093